java/semantickernel-api/src/main/java/com/microsoft/semantickernel/hooks/Hooks.java

```diff
@@ -0,0 +1,94 @@
+package com.microsoft.semantickernel.hooks;
+
+import com.microsoft.semantickernel.hooks.KernelHook.FunctionInvokedHook;
+import com.microsoft.semantickernel.hooks.KernelHook.FunctionInvokingHook;
+import com.microsoft.semantickernel.hooks.KernelHook.PreChatCompletionHook;
+import com.microsoft.semantickernel.hooks.KernelHook.PromptRenderedHook;
+import com.microsoft.semantickernel.hooks.KernelHook.PromptRenderingHook;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import java.util.function.Function;
+import javax.annotation.Nullable;
+
+public class Hooks {
```

Comment:
Should this be KernelHooks? Hook, HookEvent, KernelHook - I'd be consistent one way or the other. Everything KernelHook* or just Hook*